.travis.yml

```diff
@@ -18,10 +32,13 @@ install:
   - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then travis_retry pip install mock; fi
   - travis_retry pip install --upgrade pip setuptools wheel twine flake8
   - pip install .
+  - pip install tox
```

Comment:
nit: move up to `travis_retry pip install --upgrade pip setuptools wheel twine flake8`

Reply:
✅